src/components/atoms/TimeTable.jsx

```diff
@@ -8,14 +8,32 @@
  * - 운영시간이 하루를 넘어가는 경우: end_time에 하루를 더해서 인자로 넘깁니다.
  * - 운영시간이 하루를 넘어기지 않는 경우: start_time과 end_time을 그대로 인자로 넘깁니다.
  *
- * @param {string} start_time 운영 시작 시간 "yyyy-mm-ddThh-mm"
- * @param {string} end_time 운영 종료 시간 "yyyy-mm-ddThh-mm"
  * @param {Array} bookedTime 예약된 시간 배열 [ {start_time: "yyyy-mm-ddThh-mm", end_time: "yyyy-mm-ddThh-mm"} ]
  */
 
-const TimeTable = ({ start_time, end_time, bookedTime }) => {
-  const startTime = new Date(start_time);
-  const endTime = new Date(end_time);
+import dayjs from "dayjs";
+
+const TimeTable = ({ optime, bookedTime }) => {
+  const today = dayjs();
+
+  let start_time;
+  let end_time;
+
+  if (today.day() === 0 || today.day() === 6) {
+    start_time = optime.weekend.start;
+    end_time = optime.weekend.end;
+  } else {
+    start_time = optime.weekday.start;
+    end_time = optime.weekday.end;
+  }
```

Comment:
IIFE로 묶으세요
연관된 코드는 묶을 수록 가독성이 좋아집니다.
```suggestion
 const [start, end] = (function() {..... return [start, end]})() 
```